nbdime/webapp/package.json

```diff
@@ -16,6 +16,7 @@
     "css-loader": "^0.25.0",
     "file-loader": "^0.9.0",
     "json-loader": "^0.5.4",
+    "node-watch": "^0.4.1",
```

Comment:
After a quick google, I think this should probably be this package instead: https://www.npmjs.com/package/watch,
That is: `"watch": "^1.0.1",`.